doc/releases/quincy.rst

```diff
@@ -5,6 +5,22 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.9 Quincy
+==============
+
+This is the ninth backport (hotfix) release in the Quincy series. We recommend that all users update to this release.
+
+Notable Changes
+---------------
+
+This release fixes a critical bluestore regression in versions 17.2.8 (#63122). Users running this releas are recommended to upgrade at the earliest convenience.
```

Comment:
BlueStore

Reply:
Also, as we've recently discussed hotfix vs dot release, which dot release is this based on?  Many users will assume that the highest numbered release incorporates backports, but that seems to not be the case.  The distinction between dot releases and hotfixes is subtle.  Like, is the RGW tiering stuff in a Squid release at all yet?  It's been in main for a while.

Reply:
Fixed BlueStore

What do you propose for the second point?